srcpkgs/python3-curio/template

```diff
@@ -11,7 +11,7 @@ maintainer="Arjan Mossel <arjanmossel@gmail.com>"
 license="BSD-3-Clause"
 homepage="https://github.com/dabeaz/curio"
 distfiles="${PYPI_SITE}/c/curio/curio-${version}.tar.gz"
```

Comment:
> 10/25/2022 ***IMPORTANT NOTE*** This is the last release on pypi.  Curio
>            will no longer be making package releases.  The most recent
> 	   version can be obtained at https://github.com/dabeaz/curio.

probably want to switch the distfile to github?

Reply:
yeah, done now (I was just being lazy and postponing).